### DISCUSSION TITLE
Can I custom the color of bold text

### DISCUSSION THREAD

**Discussion by dududududv** [docsifyjs/docsify#1704] [Q&A] [answered]
Can I custom the color of bold text?
I want to set the color same as theme color.
Thanks

**Comment by trusktr** [comment, marked as answer]
You can customize the style of your site just like any other site using [CSS](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/CSS_basics).

Try putting this in your html file:

```html
<style>
  /* Any bold text will be color deeppink. */
  strong {
    color: deeppink;
  }
</style>
```

This works because the markdown outputs `<strong>` HTML elements around your bold text, so you can style those `<strong>` elements.

You can find out by right-clicking on bold text, **like this one**, then selecting the `Inspect` option, and you will see which element is used.

**Reply by dududududv** [reply]
Thanks!
